bundles/mapping/layerswipe/instance.js

```diff
@@ -39,25 +39,44 @@ Oskari.clazz.define(
 
         _startImpl: function (sandbox) {
             this.mapModule = Oskari.getSandbox().findRegisteredModuleInstance('MainMapModule');
+            this.publisherInstance = Oskari.getSandbox().findRegisteredModuleInstance('Publisher2');
```

Comment:
If layerswipe is started before publisher, this will be "undefined". We should fetch the reference when we need to do something with it to make sure it works even with funky bundle sort order.